app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/error_summary.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(errors: Seq[FormError], startAtTabIndex : Option[Int] = None)(implicit messages: Messages)
+@if(errors.nonEmpty) {
+    <div class="error-summary error-summary--show" id="error-summary" role="group" aria-labelledby="error-summary-heading" tabindex="@startAtTabIndex.map(index => index + 1)">
+
+        <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
+            @messages("error.summary.title")
+        </h2>
+
+        <ul role="list" class="error-summary-list">
+            @for((error: FormError, index: Int) <- errors.zipWithIndex) {
+                <li><a href="#@{error.key}" tabindex="@startAtTabIndex.map(x => x+2+(0*index))">@messages(error.message, error.args:_*)</a></li>
```

Comment:
0*index in last part of this will always yield 0. Might not cause an issue as you can have repeated tab indexes, but if it's not needed might as well remove it, if it is needed, should produce unique results.

Reply:
Agreed this will always map to `x+2`

Reply:
are u suggesting  then **x+ 2 + index** ?